cos-fleetshard-operator-camel/src/main/java/org/bf2/cos/fleetshard/operator/camel/CamelOperandSupport.java

```diff
@@ -365,6 +321,13 @@ public static ObjectNode createIntegrationSpec(
                 .put("value", k + "=" + v);
         });
 
+        if (!processorSpec.isEmpty()) {
```

Comment:
can we add a flag to toggle processors ?

Reply:
Done